components/openindiana/illumos-gcc/illumos-gcc.p5m

```diff
@@ -15,76 +16,76 @@
 set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
 set name=pkg.summary value="$(COMPONENT_SUMMARY)"
 set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
-set name=info.upstream-url value="$(COMPONENT_PROJECT_URL)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
 set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
 set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
-license $(COMPONENT_LICENSE_FILE) license="$(COMPONENT_LICENSE)"
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
 hardlink \
-    path=opt/gcc/$(COMPONENT_VERSION)/bin/$(GNU_ARCH)-c++ \
+    path=opt/gcc/$(COMPONENT_VERSION)/bin/i386-pc-solaris2.11-c++ \
     target=g++
 hardlink \
-    path=opt/gcc/$(COMPONENT_VERSION)/bin/$(GNU_ARCH)-g++ \
+    path=opt/gcc/$(COMPONENT_VERSION)/bin/i386-pc-solaris2.11-g++ \
     target=g++
 hardlink \
-    path=opt/gcc/$(COMPONENT_VERSION)/bin/$(GNU_ARCH)-gcc \
+    path=opt/gcc/$(COMPONENT_VERSION)/bin/i386-pc-solaris2.11-gcc \
     target=gcc
 hardlink \
-    path=opt/gcc/$(COMPONENT_VERSION)/bin/$(GNU_ARCH)-gcc-$(COMPONENT_VERSION) \
```

Comment:
I'm not sure about this. On one hand, changing manifest when nothing changed seems strange. On the other - it's good when sample-manifest and actually used manifest doesn't differ where they shouldn't. Don't have strong preference here.